eng/versioning/version_shared.py

```diff
@@ -23,6 +23,7 @@
 from common_tasks import process_glob_string, run_check_call
 
 VERSION_PY = "_version.py"
+OLD_VERSION_PY = "version.py"
```

Comment:
I'd call out in a comment that this change is ideally temporary and should be removed when code generation generates `_version.py` files.

Reply:
Agree. Ref #10355